### PR TITLE
4.5 ga fixes

### PR DESCRIPTION
**Pull request by clacroix12**
Fixes: #2390

### CODE REVIEW

File: conf/ocp_version/ocp-4.5-ga-config.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+# Config file for OCP GA 4.5 stable channel
+
+RUN:
+  client_version: '4.5-ga'
+
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.5-ga"
+  terraform_version: "0.11.14"
```
Comment (vavuthu) [inline conf/ocp_version/ocp-4.5-ga-config.yaml:10]:
```suggestion
  terraform_version: "0.12.26"
```

File: ocs_ci/utility/localstorage.py [hunk 2]
```diff
@@ -169,8 +171,29 @@ def get_lso_channel():
...
 
     """
-    channel = get_ocp_version()
-    # Workaround for https://github.com/red-hat-storage/ocs-ci/issues/2324
-    if channel == '4.5' or '4.6':
-        channel = '4.4'
-    return channel
+    ocp_version = get_ocp_version()
+    # Retrieve available channels for LSO
+    cmd = (
```
Comment (petr-balogh) [inline ocs_ci/utility/localstorage.py:176]: Why not use PackageManifest object https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/packagemanifest.py ?

Comment (clacroix12) [reply ocs_ci/utility/localstorage.py]: Good call! I totally forgot we had this class.

File: conf/ocp_version/ocp-4.5-ga-config.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.5-ga"
+  terraform_version: "0.12.26"
+  ocp_channel: "fast"
+  # No promoted content to stable-4.5 channel yet
+  # Once the build promoted to stable, we can revert back
+  # http://post-office.corp.redhat.com/archives/aos-devel/2020-May/msg00071.html
+ENV_DATA:
+  vm_template: 'rhcos-4.5.1-x86_64-vmware.x86_64'
```
Comment (vavuthu) [inline conf/ocp_version/ocp-4.5-ga-config.yaml:16]:
```suggestion
  vm_template: 'rhcos-4.5.2-x86_64-vmware.x86_64'
```

**Comment by clacroix12** [comment]
Testing is currently blocked by the issue being fixed in #2546

**Comment by clacroix12** [comment]
I have verified that the changes to `get_lso_channel` work as intended. The final value in these code blocks is the value being returned by `get_lso_channel` to be used.

When OCP version exists as channel:
```
>>> get_lso_channel()
OCP Version: 4.4
'4.4'
```
```
>>> get_lso_channel()
OCP Version: 4.5
'4.5'
```

When OCP version does not exist as channel:
```
>>> get_lso_channel()
OCP Version: 4.6
'4.5'
```

As you can see this fixes the issue of deploying a version of OCP that does not have a corresponding LSO channel yet.

**Comment by clacroix12** [comment]
I've also executed a job [here](https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/10429/) which used the the 4.5 GA configuration, further verifying this PR.

**Pull request merged**